networking_cisco/plugins/cisco/db/l3/l3_router_appliance_gbp_db.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2014 Cisco Systems, Inc.  All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from networking_cisco.plugins.cisco.db.l3 import (
+    l3_router_appliance_db as l3_r_db)
+from networking_cisco.plugins.cisco.l3.drivers import driver_context
+
+from neutron.common import constants as l3_constants
+from neutron.common import exceptions as n_exc
+from neutron.i18n import _LW
+
+from oslo_config import cfg
+from oslo_log import log as logging
+
+FLOATINGIP_STATUS_ACTIVE = l3_constants.FLOATINGIP_STATUS_ACTIVE
+
+LOG = logging.getLogger(__name__)
+
+
+class L3RouterApplianceGbpDBMixin(l3_r_db.L3RouterApplianceDBMixin):
```

Comment:
Good point. I'll add some.

On Thu, May 19, 2016 at 1:25 PM, kentwu1 <notifications@github.com> wrote:

> In networking_cisco/plugins/cisco/db/l3/l3_router_appliance_gbp_db.py
> <https://github.com/CiscoSystems/networking-cisco/pull/28#discussion_r63920044>
> :
>
> > +    l3_router_appliance_db as l3_r_db)
> > +from networking_cisco.plugins.cisco.l3.drivers import driver_context
> > +
> > +from neutron.common import constants as l3_constants
> > +from neutron.common import exceptions as n_exc
> > +from neutron.i18n import _LW
> > +
> > +from oslo_config import cfg
> > +from oslo_log import log as logging
> > +
> > +FLOATINGIP_STATUS_ACTIVE = l3_constants.FLOATINGIP_STATUS_ACTIVE
> > +
> > +LOG = logging.getLogger(__name__)
> > +
> > +
> > +class L3RouterApplianceGbpDBMixin(l3_r_db.L3RouterApplianceDBMixin):
>
> no UTs created for patch?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/CiscoSystems/networking-cisco/pull/28/files/7baa142c43ed0ed8c357e02f2aa374c0c14a7468#r63920044>
>